### DISCUSSION TITLE
myfreemp3juices.cc not loading when searching, how to fix?

### DISCUSSION THREAD

**Discussion by goodstuff626** [uBlockOrigin/uAssets#16626] [General]
When searching on myfreemp3juices.cc it usually does not load, and just hangs with ubo turned on. (sometimes it will work, but not usually) When I turn it off, it always loads good, but I get tones of pop ups. Can any advanced user tell me what I can do to fix this problem? Is there a filter i can add that can fix the problem? I'm only using the filter lists that come with ubo, no added custom filters or anything, and chrome browser. Please and thanks for any help.

**Comment by stephenhawk8054** [comment]
I can search normally without hanging. Can you click on [Report button (chat icon)](https://github.com/gorhill/uBlock/wiki/Quick-guide:-popup-user-interface#the-tools) and copy your `Troubleshooting Information` to here?

**Reply by goodstuff626** [reply]
Hey thanks for reaching out. Here is the troubleshooting info.

```yaml
uBlock Origin: 1.46.0
Chromium: 109
filterset (summary): 
  network: 140814
  cosmetic: 216135
  scriptlet: 37614
  html: 0
listset (total-discarded, last-updated): 
  added: 
    adguard-annoyance: 63328-96, 2d.18h.36m
    adguard-generic: 68215-534, 2d.18h.35m
    adguard-social: 19818-835, 2d.18h.35m
    adguard-spyware-url: 897-2, 2d.18h.35m
    block-lan: 44-0, 2d.18h.35m
    curben-phishing: 26317-0, 18h.31m
    curben-pup: 191-0, 18h.30m
    fanboy-annoyance: 84730-6404, 2d.18h.35m
    fanboy-cookiemonster: 45265-45265, 2d.18h.35m
    fanboy-social: 23040-23040, 2d.18h.35m
    fanboy-thirdparty_social: 69-20, 2d.18h.35m
    ublock-annoyances: 4794-243, 2d.18h.35m
  default: 
    user-filters: 32-0, never
    easylist: 60983-842, 2d.18h.35m
    easyprivacy: 30913-1369, 2d.18h.35m
    plowe-0: 3670-912, 2d.18h.35m
    ublock-abuse: 77-0, 2d.18h.35m
    ublock-badware: 4216-312, 2d.18h.35m
    ublock-filters: 32263-3916, 2d.18h.35m
    ublock-privacy: 259-10, 2d.18h.35m
    ublock-quick-fixes: 535-163, 18h.29m
    ublock-unbreak: 1870-95, 2d.18h.35m
    urlhaus-1: 7428-3, 18h.28m
filterset (user): [array of 32 redacted]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 597 ms (selfie)
  maxAssetCacheWait: 1075 ms
popupPanel: 
  blocked: 5
  blockedDetails: 
    brocardcored.com: 1
    googletagmanager.com: 1
    iclickcdn.com: 1
    jeeryzest.com: 1
    skimresources.com: 1
```

**Comment by mapx-** [comment]
I disabled completely uBO from extensions page, and still can reproduce the bad behaviour.

In console:

![image](https://user-images.githubusercontent.com/10303732/216792251-d8c9b1bb-90d1-4302-b5b8-2fc42b2073ee.png)

**Reply by goodstuff626** [reply]
You are right. i just tested as well with ubo turned off and and after a couple searches it started to hang. I swear, a few days ago i searched over 10 times and it worked everytime with ubo turned off. I guess it's just a buggy website. thanks for the help everyone.

**Comment by uBlock-user** [comment]
Site is broken.
